package.json

```diff
@@ -45,9 +45,9 @@
   },
   "bin": {
     "sl-pm": "./bin/sl-pm.js",
-    "sl-pmctl": "./bin/sl-pmctl.js",
+    "sl-pmctl": "./node_modules/strong-mesh-models/bin/sl-meshctl.js",
```

Comment:
This is fragile, it assumes s-m-m is installed as a direct dep of strong-pm, which isn't required, if strong-pm and strong-scheduler were both deps of strongloop, for example, s-m-m might be installed at the root.

Its also backards incompatible with strongloop, which relies on bin/sl-pm.js existing.

I think leaving bin-sl-pm.js in place, with a content of
```
require('strong-mesh-models/bin/sl-meshctl')
```

Is more robust to npm's module placement algorithm, and avoids having to bump pm to v4.x.